### DISCUSSION TITLE
lldpd daemon doesn't start on redhat 7

### DISCUSSION THREAD

**Discussion by Hugoboubou9** [lldpd/lldpd#509] [Q&A]
I would like to start lldpd daemon to have acces at "lldpcli show neighbors" command.

But, as you can see, daemon doesn't start and i can't fix it. Could you help me please ? :D

This is what i have : 
```
[root@toto]# systemctl start lldpd
Job for lldpd.service failed because the control process exited with error code. See "systemctl status lldpd.service" and "journalctl -xe" for details.

[root@toto]# systemctl status lldpd


● lldpd.service - LLDP daemon
   Loaded: loaded (/usr/lib/systemd/system/lldpd.service; enabled; vendor preset: disabled)
   Active: failed (Result: start-limit) since Wed 2022-04-20 10:20:30 CEST; 5s ago
     Docs: man:lldpd(8)
  Process: 32032 ExecStart=/usr/sbin/lldpd $DAEMON_ARGS $LLDPD_OPTIONS (code=exited, status=1/FAILURE)
 Main PID: 32032 (code=exited, status=1/FAILURE)

Apr 20 10:20:30 toto.com systemd[1]: Failed to start LLDP daemon.
Apr 20 10:20:30 toto.com systemd[1]: Unit lldpd.service entered failed state.
Apr 20 10:20:30 toto.com systemd[1]: lldpd.service failed.
Apr 20 10:20:30 toto.com systemd[1]: lldpd.service holdoff time over, scheduling restart.
Apr 20 10:20:30 toto.com systemd[1]: Stopped LLDP daemon.
Apr 20 10:20:30 toto.com systemd[1]: start request repeated too quickly for lldpd.service
Apr 20 10:20:30 toto.com systemd[1]: Failed to start LLDP daemon.
Apr 20 10:20:30 toto.com systemd[1]: Unit lldpd.service entered failed state.
Apr 20 10:20:30 toto.com systemd[1]: lldpd.service failed.
```

**Comment by vincentbernat** [comment]
What's the output of `journalctl -u lldpd`?

**Comment by Hugoboubou9** [comment]
[root@toto etc]# journalctl -u lldpd
-- Logs begin at Thu 2022-04-07 07:46:32 CEST, end at Wed 2022-04-20 11:28:42 CEST. --
Apr 14 16:32:30 toto.com systemd[1]: Starting LLDP daemon...
Apr 14 16:32:31 toto.com systemd[1]: Started LLDP daemon.
Apr 14 16:32:31 toto.com lldpd[17812]: no libcap support, running monitor as root
Apr 14 16:32:31 toto.com lldpd[17814]: /etc/localtime copied to chroot
Apr 14 16:32:31 toto.com lldpd[17814]: protocol LLDP enabled
Apr 14 16:32:31 toto.com lldpd[17814]: protocol CDPv1 disabled
Apr 14 16:32:31 toto.com lldpd[17814]: protocol CDPv2 disabled
Apr 14 16:32:31 toto.com lldpd[17814]: protocol SONMP disabled
Apr 14 16:32:31 toto.com lldpd[17814]: protocol EDP disabled
Apr 14 16:32:31 toto.com lldpd[17814]: protocol FDP disabled
Apr 14 16:32:31 toto.com lldpd[17814]: libevent 2.0.21-stable initialized with epoll method
Apr 14 16:32:31 toto.com lldpd[17814]: cannot get ethtool link information with GLINKSETTINGS (requires 4.9+). 25G+ speeds may be missing in MAC/PHY TLVs
Apr 14 16:32:31 toto.com lldpcli[17813]: lldpd should resume operations
Apr 14 16:38:48 toto.com systemd[1]: Stopping LLDP daemon...
Apr 14 16:38:48 toto.com systemd[1]: Stopped LLDP daemon.
Apr 20 09:27:11 toto.com systemd[1]: Starting LLDP daemon...
Apr 20 09:27:11 toto.com lldpd[26895]: unable to create control socket because it already exists
Apr 20 09:27:11 toto.com lldpd[26895]: check if another instance is running
Apr 20 09:27:11 toto.com lldpd[26895]: another instance is running, please stop it
Apr 20 09:27:11 toto.com lldpd[26895]: giving up
Apr 20 09:27:11 toto.com systemd[1]: lldpd.service: main process exited, code=exited, status=1/FAILURE
Apr 20 09:27:11 toto.com systemd[1]: Failed to start LLDP daemon.
Apr 20 09:27:11 toto.com systemd[1]: Unit lldpd.service entered failed state.
Apr 20 09:27:11 toto.com systemd[1]: lldpd.service failed.
Apr 20 09:27:11 toto.com systemd[1]: lldpd.service holdoff time over, scheduling restart.
Apr 20 09:27:11 toto.com systemd[1]: Stopped LLDP daemon.
Apr 20 09:27:11 toto.com systemd[1]: Starting LLDP daemon...
Apr 20 09:27:11 toto.com lldpd[26903]: unable to create control socket because it already exists
Apr 20 09:27:11 toto.com lldpd[26903]: check if another instance is running
Apr 20 09:27:11 toto.com lldpd[26903]: another instance is running, please stop it
Apr 20 09:27:11 toto.com lldpd[26903]: giving up
Apr 20 09:27:11 toto.com systemd[1]: lldpd.service: main process exited, code=exited, status=1/FAILURE
Apr 20 09:27:11 toto.com systemd[1]: Failed to start LLDP daemon.
Apr 20 09:27:11 toto.com systemd[1]: Unit lldpd.service entered failed state.
Apr 20 09:27:11 toto.com systemd[1]: lldpd.service failed.
Apr 20 09:27:11 toto.com systemd[1]: lldpd.service holdoff time over, scheduling restart.
Apr 20 09:27:11 toto.com systemd[1]: Stopped LLDP daemon.
Apr 20 09:27:11 toto.com systemd[1]: Starting LLDP daemon...
Apr 20 09:27:11 toto.com lldpd[26907]: unable to create control socket because it already exists
Apr 20 09:27:11 toto.com lldpd[26907]: check if another instance is running
Apr 20 09:27:11 toto.com lldpd[26907]: another instance is running, please stop it
Apr 20 09:27:11 toto.com lldpd[26907]: giving up
Apr 20 09:27:11toto.com systemd[1]: lldpd.service: main process exited, code=exited, status=1/FAILURE
Apr 20 09:27:11 toto.com systemd[1]: Failed to start LLDP daemon.
Apr 20 09:27:11 toto.com systemd[1]: Unit lldpd.service entered failed state.
Apr 20 09:27:11 toto.com systemd[1]: lldpd.service failed.
Apr 20 09:27:11 toto.com systemd[1]: lldpd.service holdoff time over, scheduling restart.
Apr 20 09:27:11 toto.com systemd[1]: Stopped LLDP daemon.
Apr 20 09:27:11 toto.com systemd[1]: Starting LLDP daemon...
Apr 20 09:27:11 toto.com lldpd[26911]: unable to create control socket because it already exists
Apr 20 09:27:11 toto.com lldpd[26911]: check if another instance is running
Apr 20 09:27:11 toto.com lldpd[26911]: another instance is running, please stop it
Apr 20 09:27:11 toto.com lldpd[26911]: giving up

**Reply by vincentbernat** [reply]
What's the output of `ps auxww | grep lldpd`? Which version are you running?

**Comment by Hugoboubou9** [comment]
[root@toto etc]# ps auxww | grep lldpd
root      7055  0.0  0.0 118852  2324 ?        Ss   Apr14   0:00 lldpd
lldpd     7057  0.0  0.0 116776  1836 ?        S    Apr14   0:02 lldpd
root     19210  0.0  0.0 112816   972 pts/0    S+   13:39   0:00 grep --color=auto lldpd



I installed this package : lldpd-1.0.4-1.el7.x86_64.rpm on redhat 7.9

**Reply by vincentbernat** [reply]
So, it's running. Maybe started manually since it's not tracked by systemd.

**Reply by Hugoboubou9** [reply]
manually ? that is to say ? ( sorry i'm a beginner )

**Reply by vincentbernat** [reply]
`kill 7055`, then start it again using `systemctl`.

**Comment by Hugoboubou9** [comment]
Ok ! it works :D a lot of thx !! 

[root@toto ~]# systemctl status lldpd
● lldpd.service - LLDP daemon
   Loaded: loaded (/usr/lib/systemd/system/lldpd.service; enabled; vendor preset: disabled)
   Active: active (running) since Wed 2022-04-20 15:08:21 CEST; 10s ago
     Docs: man:lldpd(8)
 Main PID: 27847 (lldpd)
   CGroup: /system.slice/lldpd.service
           ├─27847 /usr/sbin/lldpd
           └─27849 /usr/sbin/lldpd

Apr 20 15:08:21 toto.com systemd[1]: Started LLDP daemon.
Apr 20 15:08:21 toto.com lldpd[27847]: no libcap support, running monitor as root
Apr 20 15:08:21 toto.com lldpd[27849]: protocol LLDP enabled
Apr 20 15:08:21 toto.com lldpd[27849]: protocol CDPv1 disabled
Apr 20 15:08:21 toto.com lldpd[27849]: protocol CDPv2 disabled
Apr 20 15:08:21 toto.com lldpd[27849]: protocol SONMP disabled
Apr 20 15:08:21 toto.com lldpd[27849]: protocol EDP disabled
Apr 20 15:08:21 toto.com lldpd[27849]: protocol FDP disabled
Apr 20 15:08:21 toto.com lldpd[27849]: libevent 2.0.21-stable initialized with epoll method
Apr 20 15:08:21 toto.com lldpd[27849]: cannot get ethtool link information with GLINKSETTINGS (requires 4.9+). 25G+ speeds may be missing in MAC/PHY TLVs


The last line is problematic to achieve my goal but is a nice start :D
